gh.go

```diff
@@ -49,18 +49,25 @@ func searchGHIssue(name string) (*github.Issue, error){
 	found, _, err := ghClient.Search.Issues(context.TODO(), "state:open repo:"+ghOwnerRepo+" "+name, &github.SearchOptions{
```

Comment:
> The query contains one or more search keywords and qualifiers. Qualifiers allow you to limit your search to specific areas of GitHub. The REST API supports the same qualifiers as GitHub.com.

https://developer.github.com/v3/search/#search-issues-and-pull-requests

> in:title - e.g. warning in:title matches issues with "warning" in their title.

https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments
```suggestion
	found, _, err := ghClient.Search.Issues(context.TODO(), "in:title state:open repo:"+ghOwnerRepo+" "+name, &github.SearchOptions{
```

Reply:
:+1: This will help narrow down the issues but it wouldn't strictly match the issue title with the `go-version` string, e.g.: it would also match titles like "problems with `go-version`"